### ISSUE TITLE
Does not call resolveId for entrypoint

### ISSUE THREAD

**Issue by CodeWitchBella** [PepsRyuu/nollup#33]
Given example from rollup docs

```
function myExample () {
    return {
      name: 'my-example', // this name will show up in warnings and errors
        resolveId ( source ) {
            if (source === 'virtual-module') {
            return source; // this signals that rollup should not ask other plugins or check the file system to find this id
            }
            return null; // other ids should be handled as usually
        },
        load ( id ) {
            if (id === 'virtual-module') {
            return 'export default "This is virtual!"'; // the source code for "virtual-module"
            }
            return null; // other ids should be handled as usually
        }
    };
}

module.exports = ({
    input: 'virtual-module', // resolved by our plugin
    plugins: [myExample()],
    output: [{
        file: 'bundle.js',
        format: 'esm'
    }]
});
```

nollup fails because it does not call resolveId for entrypoint

This is low-priority issue because it is really easy to work around but i figured it would be good to report.

**Comment by piotr-cz** [comment]
I cannot make nollup work with @rollup/plugin-virtual in [Using the Plugin for Bundle Input](https://github.com/rollup/plugins/tree/master/packages/virtual#using-the-plugin-for-bundle-input) mode, this may be related

**Comment by PepsRyuu** [comment]
Will be fixed in the next release ```0.13.3```
